kubernetes-addons/memory-bandwidth-exporter/collector/class_clollector.go

```diff
@@ -0,0 +1,188 @@
+package collector
```

Comment:
Perhaps want to fix this file name .. guessing you intended it to be "class_collector.go

Reply:
Sorry for my carelessness, I have fixed class_collector file name.